### DISCUSSION TITLE
No Github Pro With Education Status

### DISCUSSION THREAD

**Discussion by necm1** [community/community#44081] [GitHub Education]
Hi!

I applied for Github Education on Oct 7, 2022 & had the Pro upgrade.
![Verified Status](https://i.imgur.com/lEwl6GM.png)

Now it looks like they removed it & want me to subscribe to it, even with my verified status. 
![Upgrade](https://i.imgur.com/WeLjFAb.png)

Anyone else facing this issue?

**Comment by samus-aran** [comment]
Hi @necm1 👋 

Apologies for the inconvenience. We've reach out to the team and will provide an update when available.

**Reply by necm1** [reply]
Hi @samus-aran!

I'm glad to hear that. I also tried to contact Github Support by making an ticket and they just redirected me to this Education Discussion Board.

Hopefully they'll fix that ASAP.

**Reply by samus-aran** [reply]
Hi @necm1 

There are a couple of options here to re-instate your Pro subscription as a student. 

1. If this downgrade happened within the past month you should be able to undo the subscription change (via your Billing and plans tab). Look to the bottom of the new subscription plan. It should give you an option to cancel your recent changes and revert your plan back to the student Pro subscription. This only works if the changes haven't exceeded the next billing date. If it has then the other options below would be better.
![billing-testing](https://user-images.githubusercontent.com/4041275/212570739-915bff8c-f78f-4d4c-aad7-db4a78df84c7.png)
2. Open up a[ billing and payments support](https://support.github.com/contact/account?tags=hubberfy_billing_and_payments) ticket and they will be able to re-instate you
3. As you are a verified student if you do [upgrade your personal plan](https://docs.github.com/en/billing/managing-billing-for-your-github-account/upgrading-your-github-subscription) then your student status offsets the costs of having a Pro Subscription.

Hope this all helps and let us know how you get on 👍
